templates/Aardvark.Template.Rendering.FSharp/OpenGL/Aardvark.Template.Rendering.OpenGL.fsproj

```diff
@@ -2,7 +2,6 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.0</TargetFramework>
-    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
i made an issue for this one. https://github.com/aardvark-community/aardvark.templates/issues/4